src/styles/index.scss

```diff
@@ -1,3 +1,57 @@
-body {
+@import 'vars';
+@import 'mixins';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+
+  @include flex-block;
```

Comment:
It's not clear what `flex-block` mixin does. It would be better to make the mixin name more descriptive. Remember that code should be self-explanatory.